src/app/components/layout/layout.component.ts

```diff
@@ -28,27 +27,17 @@ export class LayoutComponent implements OnInit {
     private tree: ElementRef;
 
     @Input()
-    private treeSize = 0.35;
+    private treeSize = .2;
```

Comment:
Would it be possible to have a max-width for the file tree? I messed with the CSS a little and failed to produce any desirable results.